### PR TITLE
Add some simple documentation

### PR DESCRIPTION
**Pull request by SimplyCmd**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,121 +1,674 @@
...
- c. Affirmer disclaims responsibility for clearing rights of other persons
-    that may apply to the Work or any use thereof, including without
-    limitation any person's Copyright and Related Rights in the Work.
-    Further, Affirmer disclaims responsibility for obtaining any necessary
-    consents, permissions or other rights required for any use of the
-    Work.
- d. Affirmer understands and acknowledges that Creative Commons is not a
-    party to this document and has no duty or obligation with respect to
-    this CC0 or use of the Work.
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (wafflecoffee) [inline LICENSE:1]: GPL is not a valid Minecraft mod license. See https://www.gnu.org/licenses/gpl-faq.html#GPLPlugins

Comment (wafflecoffee) [reply LICENSE:1]: Besides, it shouldn't be put in this pull request. This will remain CC0/Public Domain like the original example mod.

**Review by wafflecoffee — CHANGES_REQUESTED** [review]
Besides the license thing, LGTM

File: LICENSE (hunk 1 above)
Comment (SimplyCmd) [reply LICENSE:1]: I didn't think it would PR that change as well, as I was planning to use my fork for personal use. I'll revert it, sorry about that!

**Review by wafflecoffee — APPROVED** [review]

**Pull request merged**